lxd/storage/drivers/utils.go

```diff
@@ -911,3 +912,15 @@ func ResolveServerName(serverName string) (string, error) {
 
 	return hostname, nil
 }
+
+// remoteVolumeMapLock acquires a lock used when mapping or unmapping remote
+// storage volumes. This lock prevents conflicts between operations trying to
+// associate or disassociate volumes with the LXD host. If the lock is
+// successfully acquired, unlock function is returned.
+func remoteVolumeMapLock(connectorName string, driverName string) (locking.UnlockFunc, error) {
+	l := logger.AddContext(logger.Ctx{"connector": connectorName, "driver": driverName})
+	l.Debug("Acquiring lock for storage connector")
+	defer l.Debug("Lock acquired for storage connector")
+
+	return locking.Lock(context.TODO(), fmt.Sprintf("StorageConnector_%s_%s", connectorName, driverName))
```

Comment:
Can the logging and lock name be updated to reflect the new function name please.